package-build.yml

```diff
@@ -0,0 +1,29 @@
+apiVersion: kctrl.carvel.dev/v1alpha1
+kind: PackageBuild
+metadata:
+  name: secretgen-controller.carvel.dev
+spec:
+  release:
+  - resource: {}
+  template:
+    spec:
+      app:
+        spec:
+          deploy:
+          - kapp: {}
+          template:
+          - ytt:
+              paths:
+              - config/package-bundle/config
+              - config/release
+          - kbld:
+              paths:
+              - '-'
+              - config/release
+      export:
+      - imgpkgBundle:
+          image: ghcr.io/vmware-tanzu/carvel-secretgen-controller
```

Comment:
Is there a way for this value to come from the release values  schema?  Seems like now we have some duplication?

https://github.com/vmware-tanzu/carvel-secretgen-controller/blob/develop/config/release/values-schema.yml

Reply:
So actually Neil this is how Package authoring CLI works as of now it reads whatever there is in `package-build.yml` and this file gets automatically generated when we do `kctrl pkg init` and during `kctrl pkg release` whatever registry URL inputted by the user it get store here.